spec/features/bucket_lists/user_creates_bucket_list_spec.rb

```diff
@@ -37,4 +37,17 @@
     expect(page).to have_content("Email")
     expect(page).to have_content("Password")
   end
+
+
+    scenario "authenticated user fails to sucessfully create a bucket list" do
+      visit new_user_session_path
+      sign_in
+      visit root_path
+      click_link 'New Bucket List'
+      fill_in 'Description', with: 'Europe'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.